docs/scos/dev/migration-program/migration-to-paas/paas-migration-documents/upgrade-project-packages.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Upgrade project packages
+description: This document describes how to upgrade project packages.
+template: howto-guide-template
+---
+
+# Upgrade project packages
+
+{% info_block infoBox %}
+
+## Resources for assessment Backend
+
+{% endinfo_block %}
+
+## Project with feature packages
+1. Open a public demo shop that has been used for your project as a boilerplate e.g. [B2C Demo Shop](https://github.com/spryker-shop/b2c-demo-shop/blob/202108.0/composer.json).
+    Change the version of the release tag to the one you are going to update.
+2. Find the feature package which you want to update from release A to release B and check if this feature package is
+    still present in release B.
+    * If it is, then change the version in `composer.json` for this feature package to newer and run the update:
+        ```bash
+        composer update spryker-feature/catalog
```

Comment:
Does it make sense to just update with dependencies right away? Instead of updating without them, getting an error and only then updating with them